app/scripts/use-snow.js

```diff
@@ -12,16 +12,45 @@ Changing this code must be done cautiously to avoid breaking the app!
   const isWorker = !self.document;
   const msg =
     'Snow detected a new realm creation attempt in MetaMask. Performing scuttling on new realm.';
+
+  // eslint-disable-next-line no-undef
+  const chromeExtensionId = chrome && chrome.runtime && chrome.runtime.id;
+  const tamedFetch = (path) => {
+    const regex = new RegExp(
+      `^chrome-extension://${chromeExtensionId}/.*?/images/.*`,
+      'u',
+    );
+    if (regex.test(path)) {
+      return fetch(path);
+    }
+    throw new Error(
+      'Unscuttled fetch can only be used on images within the extension',
```

Comment:
Unscuttled -> tamed

Reply:
Will do. Can you explain that suggestion a bit? I am still building up my understanding of how scuttiling works, and I suspect your suggestion here is meaningful and I could learn something by hearing your thoughts on this suggestion

Reply:
Yes of course, gladly.
"Unscuttled" is correct, because it is in fact going to be exempt from the scuttling process.
But it's not accurate enough, because "Unscuttled" really reminds something that is being completely over looked, which isn't the case here.
`fetch` is going to be "Unscuttled" only because we promise to tame it - that is the only legitimate excuse for excepting it from being scuttled.
That is why I think referring to it as tamed rather than unscuttled is important here - so it's clear that we mutated it in order to justify it from not being scuttled.

Reply:
done in 1dd34f9af5